### PR TITLE
Scripts/Commands: Allow .cheat god to target other players

### PR DESCRIPTION
**Pull request by Artamedes**
**Changes proposed:**

-  Allow cheat god to target other players, useful for testing bosses without other gm chars

**Target branch(es):** 3.3.5/master

- [x] 3.3.5

**Tests performed:** (Does it build, tested in-game, etc.)
Builds, works ingame

**Known issues and TODO list:** (add/remove lines as needed)

- [x] must be test!

### CODE REVIEW

**Comment by jackpoz** [comment]
This PR will allow any GM to make their friends invincible. Please consider putting it in a RBAC disabled by default

File: src/server/game/Handlers/MovementHandler.cpp [hunk 1]
```diff
@@ -204,6 +210,27 @@ void WorldSession::HandleMoveWorldportAck()
 
     //lets process all delayed operations on successful teleport
     player->ProcessDelayedOperations();
+
+    if (player->GetCommandStatus(CHEAT_GOD) && !HasPermission(rbac::RBAC_PERM_COMMAND_CHEAT_GOD))
```
Comment (jackpoz) [inline src/server/game/Handlers/MovementHandler.cpp]: what does this part actually do ? it wasn't included in the PR description

**Comment by Shauren** [comment]
I don't think this is a good idea

**Comment by jackpoz** [comment]
Something I often use when I don't want to die but don't want to use .cheat god is https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=30839 together with https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?item=12947

**Comment by jackpoz** [comment]
Hi, thanks for the PR. Even if these changes can help a lot test instances, I'm not still convinced that we want to have players running around with ".cheat god", nor have some logs that iterate all the current online players.
I would prefer to see the issue that this PR targets to be fixed in a different way.

**Pull request closed without merging**